docs/modules/operation/pages/deep-dive/elasticsearch/introduction.adoc

```diff
@@ -164,3 +182,9 @@ Credentials for each node must be stored in `$OPENNMS_HOME/etc/elastic-credentia
 ----
 
 NOTE: Credentials are globally defined and will be used by all features that connect to the specified URLs.
+
+== Circuit breaker
+
+In the case when Elasticsearch is unavailable, a circuit breaker is used to reduce the amount of blocked threads. After a defined number of errors, requests to the server will not be sent for defined time. Instead, the requests are aborted immediately with an error. To configure the circuit breaker, use the parameters that begin with `circuitbreaker.`.
```

Comment:
```suggestion
When Elasticsearch is unavailable, a circuit breaker is used to reduce the number of blocked threads.
After a defined number of errors, requests to the server will not be sent for the defined period of time.
Instead, the requests are cancelled immediately with an error.
To configure the circuit breaker, use the parameters that begin with `circuitbreaker.`.
```
Does it make sense to list the defaults for the defined number of errors and defined time?

After a defined number of errors (by default X), requests to the server will not be sent for the defined period of time (by default, Y).